plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -59,15 +62,12 @@ protected void execute(IProgressMonitor monitor)
     properties.put("application-id", appId);
 
     ProjectImportConfiguration importConfiguration = new ProjectImportConfiguration();
-    List<IProject> archetypeProjects = projectConfigurationManager
-        .createArchetypeProjects(location, getArchetypeDescriptor(), groupId, artifactId,
-            "0.0.1-SNAPSHOT",
-            packageName, properties, importConfiguration, progress.newChild(60));
-
-    if (!archetypeProjects.isEmpty()) {
-      Job job = new MappingDiscoveryJob(archetypeProjects);
-      job.schedule();
-    }
+    List<IProject> archetypeProjects = projectConfigurationManager.createArchetypeProjects(location,
+        getArchetypeDescriptor(), groupId, artifactId, version, packageName, properties,
+        importConfiguration, progress.newChild(60));
+
+    Job job = new MappingDiscoveryJob(archetypeProjects);
```

Comment:
Noticed that this class is not API. We should figure out if that's a problem. Maybe add a TODO here.